addons/pointwatch/statics.lua

```diff
@@ -142,7 +142,7 @@ function initialize()
         setfenv(cur_func,_G)
         dynamis.entry_time = os.clock()
         dynamis.zone = info.zone
-        windower.add_to_chat(123,'Loading PointWatch in Dynamis results in an inaccurate timer. Number of KIs is displayed.')
+        error(123,'Loading PointWatch in Dynamis results in an inaccurate timer. Number of KIs is displayed.')
```

Comment:
The `123` needs to be removed for this to work right. Was this change intended? This doesn't sound like an error.